src/Hit/Git.hs

```diff
@@ -73,6 +73,14 @@ runNew issueOrName = do
                        || isSpace c
                        || c `elem` ("_-./" :: String)
                    )
+runNew _ title =
+    createIssue' title >>= \case
+        Left err -> errorMessage $ show err
+        Right issue -> do
+          putTextLn . showIssueName blueCode 0 $ issue
+          runStash
+          "git" ["checkout", "master"]
+          runNew False $ show $ unIssueNumber . issueNumber $ issue
```

Comment:
I have a look at https://github.com/phadej/github but cannot find the API to assign a user to an issue. Hence, I leave it out at the moment.
Please let me know if you have any idea to do it.

Reply:
@bangn I think it's okay to leave it as it is at the moment. We will try to investigate `github` library and see what can be done.